### ISSUE TITLE
AI drop downs changing between battles

### ISSUE THREAD

**Issue by Xendergo** [treeform/istrolid-bugs#66]
AI drop downs changing between battles

I have an AI with a kite command, and between battles, it switches to attack for no clear reason.

**Comment by bokken12** [comment]
Could you give more detailed steps to replicate your issue? I have many AIs with the kite command, and while I don't deny that this is a problem, not all of them do so.

Which command specifically did you set to kite? What attack type was it using previously? Did you go directly to a battle afterwards? How did you leave the design screen? Was the change only visual or did it affect the performance of the AI?

**Comment by Xendergo** [comment]
I don't know exactly what caused it, so I can't give more detailed instructions to reproduce it.

____enemy that is ____ and ____ within ____m

I don't know what you mean

I went directly into the server, not the battle

I don't remember

It affected the AI

I _**THINK**_ it might have something to do with #65

**Comment by bokken12** [comment]
I don't believe the bugs are related, my best guess at the moment is simply that it failed to save. Ships are saved whenever a new part is added to them, however when an AI dropdown is changed they are not. I've made a commit to change this that could be worth testing however it is not yet in master.
